src/racingcar/main.py

```
@@ -1,12 +1,53 @@
+        raise ValueError("자동차 이름은 1~5자의 문자여야 합니다.")
+
+    return names
+
+def get_attempt_count():
+    try:
+        count = int(input("시도할 횟수는 몇 회인가요?\n"))
+        if count <= 0:
+            raise ValueError
+        return count
+    except ValueError:
+        raise ValueError("올바른 횟수를 입력하세요. (양의 정수)")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)